Templates/WebHook-Generic-CSharp/run.csx

```diff
@@ -0,0 +1,25 @@
+﻿#r "Newtonsoft.Json"
+
+using System;
+using System.Net;
+using System.Net.Http;
+using Microsoft.Azure.WebJobs.Host;
+using Newtonsoft.Json;
+
+public static object Run(HttpRequestMessage req, TraceWriter log)
+{
+    log.Verbose($"Webhook was triggered!");
+
+    string jsonContent = req.Content.ReadAsStringAsync().Result;
```

Comment:
I think you can do something like `dynamic data = await response.Content.ReadAsAsync<object>();`. Have you tried that?